concepts/comparisons/about.md

```diff
@@ -1,2 +1,184 @@
+False
+>>> 17 == 17.0
+True
+>>> 17.0 == 0017.000
+True
+>>> complex(1, 2) == complex(1.0, 2.00)
+True
+```
+
+Python makes this distinction because strings are text, while `integers` and `floats` are numeric types.
+
+## Value Comparison
+
+The operators `<`, `>`, `==`, `>=`, `<=`, and `!=` compare the _values_ of two different objects. The objects do not need to have the same type. Remember that in Python
+
+every object has a `value` in addition to `type` and `identity`.
+
+### Following are comparison behaviour of most `built-ins` types.
+
+Numbers of built-in _numeric_ types such as `int`, `hex`, `ocal`, `binary`, `float`, `complex` and of the standard library types `fractions.Fraction` and `decimal.Decimal` can be compared within and across their types.
+
+Any ordered comparison of a number to a `NaN` (_not a number_) value is `False`. A counter-intuitive implication is that `NaN` never compares equal to `NaN`.
+
+```python
+>>> x = float('NaN')
+>>> x
+nan
+>>> 3 < x
+False
+>>> x < 3
+False
+>>> x == x
+False
+```
+
+Strings (`str`) are compared _lexicographically_ using their individual numerical Unicode code points (_the result of passing each code point in the `str` to the built-in function `ord()`_). `str` and `binary` sequences cannot be directly compared.
+
+```python
+>>> 'santa' < 'claus'
+False
+>>> 'Santa' < 'claus'
+True
+>>> ord('s')
+115
+>>> ord('S')
+83
+>>> ord('c')
+99
+>>> '龙波' < '王想'  # chinese words
+False
+>>> # check ord() of first letters
+>>> ord('龙')
+40857
+>>> ord('王')
+29579
+>>>
+>>>
+>>> # let's try korean words
+>>> '이서윤' < '김은정'
+False
+# compare their first letters
+>>> ord('이') < ord('김')
+False
+```
+
+Collections like `list`, `set`, `tuple` and `dict` can also be compared -- provided they are of the same `type`, have the same length, and each _**pair**_ of corresponding elements within the collection are comparable.
+
+```python
+>>> [1, 2] == [1, 2]
+True
+>>> (1, 2) == [1, 2]
+False
+>>> [1, 2] < [1, 2, 3]
+True
+>>> # comparison of dicts
+>>> {'name': 'John', 'age': 19} == {'name': 'John', 'age': 18}
+False
+>>> {'name': 'John', 'age': 19} == {'name': 'John', 'age': 19}
+True
+```
+
+## Identity comparisons
+
+The operators `is` and `is not` test for an object's _identity_. An object's identity is determined using the `id()` function.
+
+`apple is orange` is `True` if and only if `apple` and `orange` are the same object. `apple is not orange` yields the inverse truth value.
+
+```python
+>>> my_fav_numbers = [1, 2, 3]
+>>> your_fav_numbers = my_fav_numbers
+>>> my_fav_numbers == your_fav_numbers
+True
+>>> id(my_fav_numbers)
+4462635008
+>>> id(your_fav_numbers)
+4462635008
+>>> my_fav_numbers is not your_fav_numbers
+False
+```
+
+## Membership test operations
+
+The operators `in` and `not in` test for _membership_. `fish in soup` evaluates to `True` if `fish` is a member of `soup`, and evaluates `False` otherwise. `fish not in soup` returns the negation, or _opposite of_ `fish in soup`.

```

Comment:
```suggestion
The operators `in` and `not in` test for _membership_.
`fish in soup` evaluates to `True` if `fish` is a member of `soup`, and evaluates `False` otherwise.
`fish not in soup` returns the negation, or _opposite of_ `fish in soup`.

```
As per the [build markdown][1] style guide; single sentence per line.

[1]: https://github.com/exercism/docs/blob/main/building/markdown/markdown.md